pkg/common/logger.go

```diff
@@ -18,10 +30,36 @@ func Logger(ctx context.Context) logrus.FieldLogger {
 			return logger
 		}
 	}
-	return logrus.StandardLogger()
+
+	formatter := new(logFormatter)
+	logger := logrus.New()
```

Comment:
With `logrus.StandardLogger()` I'm not able to capture the output in the tests, any ideas?

Didn't see another difference between `logrus.New()` and `logrus.StandardLogger()`...